lib/message_generator.dart

```diff
@@ -28,10 +28,17 @@ class MessageGenerator extends ProtobufContainer {
   final String classname;
 
   /// The fully-qualified name of the message type.
-  ///
-  /// (Used as a unique key and in error messages, not in Dart code.)
   final String fqname;
 
+  // TODO(sigurdm): is this the right thing?
+  String get fullName {
```

Comment:
A ternary may be cleaner here. Also, what's the special case with the leading '.'? When does that happen? I'm wondering if there's some way we can avoid needing a separate fqname and fullName.

Reply:
Tenary: yes indeed!
I also first wanted to use fqname, but it adds the leading dot. Not sure I really understand why, or if it has a meaning, but removing the dot from fqname is a bit of a refactor. I will investigate more.

Reply:
Since this is now a single statement, you can just use the => return syntax.

Nit: I believe the outer parens are unnecessary for the check.